cpp11test/src/test-as.cpp

```diff
@@ -476,9 +476,10 @@ context("as_cpp-C++") {
     UNPROTECT(1);
   }
 
-  test_that("as_sexp(r_vector<cpp11::r_string>)") {
-    SEXP s1 =
-        PROTECT(cpp11::as_sexp(std::vector<cpp11::r_string>({"foo", "bar", "baz"})));
+  test_that("as_sexp(cpp11::strings)") {
+    cpp11::writable::strings s({"foo", "bar", "baz"});
+    cpp11::strings s0(s);
+    SEXP s1 = PROTECT(cpp11::as_sexp(s0));
```

Comment:
this isn't transparent to me; did compilation break or is it just a preferred spelling of this test or ...?

Reply:
Yeah this particular case `as_sexp(cpp11::string)` is something that occurs in [vroom](https://github.com/r-lib/vroom) but we didn't use it in the cpp11test code at all, so I added a test case for it specifically.

Reply:
alright, but let's keep the original one as well
```suggestion
  test_that("as_sexp(cpp11::strings)") {
    cpp11::writable::strings s({"foo", "bar", "baz"});
    cpp11::strings s0(s);
    SEXP s1 = PROTECT(cpp11::as_sexp(s0));
    
    SEXP s2 = PROTECT(cpp11::as_sexp(std::vector<cpp11::r_string>({"foo", "bar", "baz"})));
```